### DISCUSSION TITLE
报错：start DNS server failed

### DISCUSSION THREAD

**Discussion by tecsm** [zzzgydi/clash-verge#877] [Q&A]
![photo_2023-11-02_20-07-49](https://github.com/zzzgydi/clash-verge/assets/50205866/b126241b-815f-4298-a763-d277e610a2f8)
导入机场链接，开启代理。报错，怎么解决啊？

**Comment by zzzgydi** [comment]
端口冲突吧，改端口或者把占用端口的进程干了
